package.json

```diff
@@ -56,12 +56,12 @@
     "iojs": ">= 3"
   },
   "dependencies": {
-    "phovea_core": "~2.1.2"
+    "phovea_core": "github:phovea/phovea_core#develop"
```

Comment:
When merging into `master` there should be no dependency to a `develop` branch!

Reply:
jab sorry